### PR TITLE
Remove dependencies

### PR DESCRIPTION
**Pull request by jvanderleeuw**
Ran pylint and added agpypeline library

### CODE REVIEW

File: agpypeline/transformer_class.py [hunk 1]
```diff
@@ -1,19 +1,17 @@
...
 import logging
 
 import piexif
-from terrautils.imagefile import get_epsg as tr_get_epsg, \
-                                 image_get_geobounds as tr_image_get_geobounds
-import terrautils.lemnatec
 
-import configuration
+from numpy import nan
+from osgeo import gdal
```
Comment (Chris-Schnaufer) [inline agpypeline/transformer_class.py:11]:
```suggestion
from osgeo import gdal, osr
```

File: agpypeline/transformer_class.py [hunk 2]
```diff
@@ -1,19 +1,17 @@
...
+from osgeo import osr
```
Comment (Chris-Schnaufer) [inline agpypeline/transformer_class.py:12]:
```suggestion
```

File: agpypeline/transformer_class.py [hunk 3]
```diff
@@ -151,7 +149,20 @@ def get_image_file_epsg(self, source_path: str) -> str:
             doesn't have an EPSG code
         """
         # pylint: disable=no-self-use
-        return tr_get_epsg(source_path)
+        logger = logging.getLogger(__name__)
```
Comment (Chris-Schnaufer) [inline agpypeline/transformer_class.py:152]: Logging here needs to be reconciled with the rest of the library. There are now two different approaches with this addition

File: agpypeline/transformer_class.py [hunk 4]
```diff
@@ -151,7 +149,20 @@ def get_image_file_epsg(self, source_path: str) -> str:
...
+
+        try:
+            src = gdal.Open(source_path)
+
+            proj = osr.SpatialReference(wkt=src.GetProjection())
+
+            return proj.GetAttrValue('AUTHORITY', 1)
+        # pylint: disable=broad-except
```
Comment (Chris-Schnaufer) [inline agpypeline/transformer_class.py:160]: Is this pylint comment still needed?

File: agpypeline/transformer_class.py [hunk 5]
```diff
@@ -151,7 +149,20 @@ def get_image_file_epsg(self, source_path: str) -> str:
...
+        except Exception as ex:
+            logger.warning("[get_epsg] Exception caught: %s", str(ex))
```
Comment (Chris-Schnaufer) [inline agpypeline/transformer_class.py:162]: Reporting wrong name

File: agpypeline/transformer_class.py [hunk 6]
```diff
@@ -163,7 +174,26 @@ def get_image_file_geobounds(self, source_path: str) -> list:
             is returned if the boundaries can't be determined
         """
         # pylint: disable=no-self-use
-        return tr_image_get_geobounds(source_path)
+        logger = logging.getLogger(__name__)
```
Comment (Chris-Schnaufer) [inline agpypeline/transformer_class.py:177]: Same logging comment as above

File: agpypeline/transformer_class.py [hunk 7]
```diff
@@ -163,7 +174,26 @@ def get_image_file_geobounds(self, source_path: str) -> list:
...
+
+            min_y = min(uly, lry)
+            max_y = max(uly, lry)
+            min_x = min(ulx, lrx)
+            max_x = max(ulx, lrx)
+
+            return [min_y, max_y, min_x, max_x]
+        # pylint: disable=broad-except
+        except Exception as ex:
+            logger.info("[image_get_geobounds] Exception caught: %s", str(ex))
```
Comment (Chris-Schnaufer) [inline agpypeline/transformer_class.py:193]: Wrong name used

File: agpypeline/transformer_class.py [hunk 8]
```diff
@@ -163,7 +174,26 @@ def get_image_file_geobounds(self, source_path: str) -> list:
...
+            lrx = ulx + (src.RasterXSize * xres)
+            lry = uly + (src.RasterYSize * yres)
+
+            min_y = min(uly, lry)
+            max_y = max(uly, lry)
+            min_x = min(ulx, lrx)
+            max_x = max(ulx, lrx)
+
+            return [min_y, max_y, min_x, max_x]
+        # pylint: disable=broad-except
```
Comment (Chris-Schnaufer) [inline agpypeline/transformer_class.py:191]: Same pylint comment as above

File: agpypeline/transformer_class.py [hunk 9]
```diff
@@ -1,19 +1,17 @@
...
-                                 image_get_geobounds as tr_image_get_geobounds
-import terrautils.lemnatec
 
-import configuration
+from numpy import nan
+from osgeo import gdal
+from osgeo import osr
 
-terrautils.lemnatec.SENSOR_METADATA_CACHE = os.path.dirname(os.path.realpath(__file__))
+import configuration
```
Comment (Chris-Schnaufer) [inline agpypeline/transformer_class.py:14]: This can't be here like this since it's not part of the library

**Comment by dlebauer** [comment]
from @jvanderleeuw closed since a new structure for agpypeline was used after this point. The branch there was used before for doing manual imports to look for files in the user's file system, but the new structure makes that unnecessary

**Pull request closed without merging**